### PR TITLE
Enabled emoji suggestion only right after ":" and a character to enable navigation

### LINKED ISSUE #582: Not able to navigate through the emojis using keyboard
https://www.loom.com/share/2a92f16533004eb69fa19f4f2b1ae202

bug _t

### PR DESCRIPTION
**Pull request by gaagul**
Fixes #582 

**Description**

- Fixed: emoji suggestion only right after colon and a character to enable navigation

**Checklist**

- [x] I have added the necessary label (patch/minor/major - If package publish is required).

**Reviewers**

@AbhayVAshokan _a

### CODE REVIEW

**Comment by gaagul** [comment]
@AbhayVAshokan Please verify and let me know if this is the intended behaviour.

**Comment by AbhayVAshokan** [comment]
The change in neetoPlanner can be picked up next week. It is not of the top priority. This method will be replacing the current implementation.

File: lib/components/Editor/CustomExtensions/Emoji/EmojiSuggestion/EmojiSuggestionMenu.jsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ class EmojiSuggestionMenu extends React.Component {
       return (await SearchIndex.search(this.props.query)).slice(0, 5);
     }
 
-    const defaultEmojis = ["+1", "-1", "smile", "tada", "heart"];
+    const defaultEmojis = [];
```
Comment (AbhayVAshokan) [inline lib/components/Editor/CustomExtensions/Emoji/EmojiSuggestion/EmojiSuggestionMenu.jsx:51]: We can simplify the `searchEmoji` method:

```js
  searchEmoji = async () =>
    this.props.query
      ? (await SearchIndex.search(this.props.query)).slice(0, 5)
      : [];
```

**Review by AbhayVAshokan — APPROVED** [review]
Awesome. This solves all the issues. Please make this small change and proceed to merge.

@gaagul _a

**Pull request merged**